app/src/main/java/com/rentminder/SaveButton.kt

```diff
@@ -35,33 +21,38 @@ import androidx.compose.ui.unit.dp
 fun SaveRemindButton(enabled: Boolean, inputEdited: MutableState<Boolean>) {
     val context = LocalContext.current
     val focusManager = LocalFocusManager.current
-    var count by remember { mutableStateOf(0) }
+    var saved by remember { mutableStateOf(false) }
 
     IconButton(
         onClick = {
-            if (enabled) {
-                if (count > 0) {
-                    Toast.makeText(context, "Edit Saved!", Toast.LENGTH_SHORT).show()
+            try{
+                if (enabled) {
+                    if (saved) {
+                        Toast.makeText(context, R.string.edit_saved_message, Toast.LENGTH_SHORT).show()
+                    } else {
+                        Toast.makeText(context, R.string.saved_message, Toast.LENGTH_SHORT).show()
+                    }
+                    saved = true
+                    inputEdited.value = false
+                    focusManager.clearFocus()
                 } else {
-                    Toast.makeText(context, "Saved!", Toast.LENGTH_SHORT).show()
+                    Toast.makeText(
+                        context,
+                        R.string.enter_value_message,
+                        Toast.LENGTH_SHORT
+                    ).show()
                 }
-                count++
-                inputEdited.value = false
-                focusManager.clearFocus()
-            } else {
-                Toast.makeText(
-                    context,
-                    "Please enter a value before saving or editing",
-                    Toast.LENGTH_SHORT
-                ).show()
+            } catch(e: Exception){
```

Comment:
Created a try catch to handle any errors that might be thrown as a result of the click event and displaying appropriate toasts to the user  using the conventions from:
https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements
https://kotlinlang.org/docs/exceptions.html

Reply:
This is a catch-all Exception.  Is an Exception likely to be thrown here?  If so, which one?

Logging in a catch block is good.  Also consider what to do with the exception.  
- Should the function continue as if all went well, or
- Should you re-throw the exception, to prevent the function from continuing.